source/uwp/Renderer/lib/ActionHelpers.cpp

```diff
@@ -563,22 +584,56 @@ namespace AdaptiveCards::Rendering::Uwp::ActionHelpers
         // can we do explicit check? or need to call check_pointer()?
         if (action)
         {
-            WireButtonClickToAction(button, action, renderContext);
+            if (action.Title().empty() && action.Tooltip().empty())
+            {
+                // We are in the same edge case mentioned above (column being used as a toggle button)
+                // so we need to ensure that we update our title/tooltip when our items swap visibilities
+                const auto callback = [adaptiveCardElement, button, description]()
+                {
+                    if (const auto col = adaptiveCardElement.try_as<winrt::AdaptiveColumn>())
+                    {
+                        for (auto item : col.Items())
+                        {
+                            if (const auto img = item.try_as<winrt::AdaptiveImage>())
+                            {
+                                // Swap the visibilities
+                                if (img.IsVisible())
+                                {
+                                    img.IsVisible(false);
+                                }
```

Comment:
The `SendActionEvent` gets called before we get into this lambda, so the XAML tree has already been updated with the correct visibilities, however there doesn't seem to be an easy way to get to the XAML visibility from this AdaptiveItem, so we do this manual update here to keep it in sync with the XAML tree. If there is a better way to do this please let me know!